go.mod

```diff
@@ -28,16 +31,20 @@ require (
 	github.com/rcrowley/go-metrics v0.0.0-20200313005456-10cdbea86bc0
 	github.com/rs/cors v1.7.0
 	github.com/sasha-s/go-deadlock v0.2.0
-	github.com/smartystreets/assertions v1.0.1 // indirect
 	github.com/snikch/goodman v0.0.0-20171125024755-10e37e294daa
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.7.0
 	github.com/tendermint/tm-db v0.6.4
-	golang.org/x/crypto v0.0.0-20201117144127-c1f2f97bffc9
+	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad
 	golang.org/x/exp v0.0.0-20200331195152-e8c3332aa8e5 // indirect
 	golang.org/x/net v0.0.0-20201021035429-f5854403a974
 	google.golang.org/genproto v0.0.0-20201119123407-9b1e624d6bc4 // indirect
 	google.golang.org/grpc v1.35.0
 	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
 )
+
+replace (
+	github.com/ipfs/go-ipfs => github.com/lazyledger/go-ipfs v0.7.0-lazypatch
+	github.com/lazyledger/lazyledger-core/p2p/ipld/plugin => ./p2p/ipld/plugin
```

Comment:
looks like the e2e tests do not properly work with this replace:
```
 ---> Running in 787f3370b488
go: github.com/lazyledger/lazyledger-core/p2p/ipld/plugin@v0.0.0-20210219190522-0eccfb24e2aa: parsing p2p/ipld/plugin/go.mod: open /src/tendermint/p2p/ipld/plugin/go.mod: no such file or directory
The command '/bin/sh -c go mod download' returned a non-zero code: 1
```

Reply:
This is fixed by 10f294c921f11b9859398afe7ce1c331cd3a84c2. But there is still some other problem with the e2e tests (timeout while waiting for peer to reach height 0).